ebms-send-in/src/main/kotlin/no/nav/emottak/ebms/App.kt

```diff
@@ -1,199 +1,60 @@
-                                }
-
-                            SupportedService.HarBorgerEgenandelFritak, SupportedService.HarBorgerFrikort ->
-                                timed(appMicrometerRegistry, "frikort-sporing") {
-                                    with(sendInRequest.asEIFellesFormat()) {
-                                        frikortsporring(this).let { frikortsporringResponse ->
-                                            SendInResponse(
-                                                messageId = sendInRequest.messageId,
-                                                conversationId = sendInRequest.conversationId,
-                                                addressing = sendInRequest.addressing.replyTo(
-                                                    frikortsporringResponse.eiFellesformat.mottakenhetBlokk.ebService,
-                                                    frikortsporringResponse.eiFellesformat.mottakenhetBlokk.ebAction
-                                                ),
-                                                payload = FellesFormatXmlMarshaller.marshalToByteArray(frikortsporringResponse.eiFellesformat.msgHead),
-                                                requestId = Uuid.random().toString()
-                                            )
-                                        }
-                                    }
-                                }
-
-                            SupportedService.HarBorgerFrikortMengde ->
-                                timed(appMicrometerRegistry, "frikortMengde-sporing") {
-                                    with(sendInRequest.asEIFellesFormat()) {
-                                        frikortsporringMengde(this).let { frikortsporringMengdeResponse ->
-                                            SendInResponse(
-                                                messageId = sendInRequest.messageId,
-                                                conversationId = sendInRequest.conversationId,
-                                                addressing = sendInRequest.addressing.replyTo(
-                                                    frikortsporringMengdeResponse.eiFellesformat.mottakenhetBlokk.ebService,
-                                                    frikortsporringMengdeResponse.eiFellesformat.mottakenhetBlokk.ebAction
-                                                ),
-                                                payload = FellesFormatXmlMarshaller.marshalToByteArray(frikortsporringMengdeResponse.eiFellesformat.msgHead),
-                                                requestId = Uuid.random().toString()
-                                            )
-                                        }
-                                    }
-                                }
+suspend fun ResourceScope.setupServer() {
+    val server = config().server
+    System.setProperty("io.netty.maxChunkSize", server.maxChunkSize)
```

Comment:
Hvorfor setter vi denne ? Er det noe som tilsier at det er nødvendig ?

Reply:
Jeg vet ikke, det er noe som har vært der fra før. Jeg tar den gjerne vekk - skal rydde litt mer :smile:

Reply:
Løses her: https://github.com/navikt/send-in-send-out/pull/49